app/services/stripe_service/report.rb

```diff
@@ -117,6 +235,14 @@ def create_charge
     }
   end
 
+  def cancel_charge
+    {
+      "stripe_op": "cancel_charge",
+      "transaction_id": tx.id,
+      "stripe_seller_id": stripe_account.stripe_seller_id
```

Comment:
Minor thing: we could report the `stripe_charge_id` here. We have the ID available both when starting the call and when it completes.